ksp_plugin_test/flight_plan_test.cpp

```diff
@@ -82,6 +102,95 @@ TEST_F(FlightPlanDeathTest, DestroyingFirstSegment) {
   }, "Destroying the first segment of flight plan");
 }
 
+TEST_F(FlightPlanTest, Singular) {
+  // A test mass falling from x₀ = 1 m at vanishing initial speed onto a body
+  // with gravitational parameter μ = 1 m³/s².  A singularity occurs for
+  // t² = π² x₀³ / μ.
```

Comment:
This formula doesn't match what's at line 110.  It would be good to define a const x0 for clarity.